cpp/src/DataStorm/NodeI.cpp

```diff
@@ -181,20 +209,10 @@ NodeI::createSession(
 
         unique_lock<mutex> lock(_mutex);
         session = createPublisherSessionServant(*subscriber);
```

Comment:
createPublisherSessionServant no longer handles  ObjectAdapterDestroyedException and return null, I think is simpler to let the callers of this method to handle it.